### DISCUSSION TITLE
layui3.0的用户侧技术方案是什么样呢？使用方式差别大吗？

### DISCUSSION THREAD

**Discussion by augushong** [layui/layui#2321] [讨论]
想问一下作者，3.0的用户侧的使用方式差别大吗？
看之前的资料，所提到的改动大，主要是针对框架底层的说明。
我使用了layui实现了一个curd后台，假设我完全按照layui的要求开发的话，后面升级3.0的话，变化大吗？

**Comment by sentsim** [comment]
使用差别不算太大，但不会兼容 2.x。
其实使用 2.9 开发的后台界面，基本上也可以满足 90%+ 的场景，短期内不建议等 3.0。

**Reply by augushong** [reply]
好的，感谢回答。
